API/DTOs/MatchesWebSubmissionDTO.cs

```diff
@@ -0,0 +1,21 @@
+using System.ComponentModel.DataAnnotations;
+
+namespace API.DTOs;
+
+/// <summary>
+/// Represents data used to add matches to an existing tournament
+/// </summary>
+public class MatchesWebSubmissionDTO
+{
+    /// <summary>
+    /// The Id (primary key) of the submitting User
+    /// </summary>
+    [Range(0, int.MaxValue)]
```

Comment:
Makes sense to start from 1, but I'm indifferent to this (gee, if only it was a standard practice to use `uint` for this stuff)